server/auth/types/proxy/proxy_auth.ts

```diff
@@ -73,6 +73,10 @@ export class ProxyAuthentication extends AuthenticationType {
       : false;
   }
 
+  public async supportsKeepAlive(request: OpenSearchDashboardsRequest): Promise<boolean> {
```

Comment:
Can you add a small comment here on why this defaults to true?

Reply:
Defaults to true since that is the current behavior. Resolving this comment as implementation changed.